lib/FusionInventory/Agent/Task/Inventory/Virtualization/Xen.pm

```diff
@@ -77,20 +77,44 @@ sub  _getVirtualMachines {
 
     # drop headers
     my $line  = <$handle>;
+    my @rheaders = reverse split ' ', $line;
+    my $nbHeaders = scalar @rheaders;
```

Comment:
if the goal is just to count the number of headers, reversing the list is useless:
my $nbheaders = scalar(split(' ', $line));